pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/rootpatches/Osteo2pecgsUpdates.java

```diff
@@ -30,7 +33,17 @@ public void init(Path cfgPath, Config studyCfg, Config varsCfg) {
         taskList.add(new Osteo2GermlineConsentVersion3());
         taskList.add(new Osteo2GermlinePedConsentVersion3());
 
+        SimpleActivityRevisionTask osteoPediatricConsentAndAssentVersion3 = new SimpleActivityRevisionTask();
+        taskList.add(osteoPediatricConsentAndAssentVersion3);
+
         taskList.forEach(task -> task.init(cfgPath, studyCfg, varsCfg));
+
+        try {
+            osteoPediatricConsentAndAssentVersion3.consumeArguments(
+                    new String[]{"patches/osteo-version-3-changes.conf"});
```

Comment:
Just a comment : From my understanding :
Adding incremental config changes to same config  (like patches/osteo-version-3-changes.conf)  file will not work in Dev env but will work in rest of envs because we apply All config changes at once. In Dev though as we complete each ticket (new config change), we apply in DEV.  I mean previous config changes might have been already applied in Dev.

Reply:
probably making a new patch config file and adding new entry might be workaround for Dev scenario.

Reply:
That's right.  We could decide in the future to change the behavior and incrementally apply the changes.